docs/solutions/connect_vcns_using_multiple_vnics/env-vars

```diff
@@ -0,0 +1,33 @@
+### Provider credentials
+export TF_VAR_tenancy_ocid=""
+export TF_VAR_user_ocid=""
+export TF_VAR_fingerprint=""
+export TF_VAR_private_key_path=""
+
+### Region
+export TF_VAR_region=""
+
+### AD
+export TF_VAR_AD=""
```

Comment:
Looks like this AD var is 1,2,3 (as opposed to zero-based of the full AD name). Should comment on that.